server/fishtest/templates/run_table.mak

```diff
@@ -66,7 +66,12 @@
                         <i class="fas fa-trash-alt"></i>
                       </button>
                       <div class="dropdown-menu" role="menu">
-                        <form action="/tests/delete" method="POST" style="display: inline;">
+                        <form 
+                          action="/tests/delete"
+                          method="POST"
+                          style="display: inline;"
+                          onsubmit="unfollow_run('${run[\'_id\']}'); return true;"
```

Comment:
The quote escaping doesn't work.

Reply:
> The quote escaping doesn't work
I would appreciate it if hinted I need the escaping at the first place?